src/content/decentralization/index.md

```diff
@@ -0,0 +1,329 @@
+
+## Ethereum is the perfect decentralized system! Right? {#challenges-to-ethereum}
+
+So, Ethereum is a perfect, utopian decentralized computing system, right? Not so fast.
+
+The network works, but it's still a work in progress. Also, like most decentralized systems, Ethereum prioritizes some forms of decentralization over others.
+
+Becoming even more awesome is always on the agenda, though. Here are some things the community is working towards:
+
+- **Increased [client diversity](/developers/docs/nodes-and-clients#advantages-of-different-implementations)**: Diverse software means less chance for a single bug or attack to cause a major problem.
+- **Better geographical distribution of network operators (active participants in many different countries)**: Having nodes in many different places reduces the risk of government interference, outages from regional disasters, etc.
+- **Distribution of ETH balances (already one of the most even distributions in crypto)**: Currency value is at risk for manipulation when the supply is largely controlled by a single entity or group of associated entities. In other words, a centralized group can decide how much is available. Increased distribution reduces this risk.
+- **Increased cooperation between the Ethereum community and other decentralized projects**: Ethereum is at the infrastructure level for the decentralized web. Linking other projects and building more applications increases competition, but it also increases utility.
+- **Better communication between network operators, users, and software developers**: Communication helps identify potential uses and resolve issues. Sometimes one group will have a perspective that the others did not consider.
+
+## Not all crypto and blockchain projects are decentralized {#centralized-vs-decentralized}
+
+"Blockchain/crypto" does not equal "decentralized". The fact is that most systems have aspects of centralization and decentralization. These design concepts can and do coexist.
+
+So the question becomes what types of decentralization are practical? What types of centralization do we absolutely need? As mentioned above, Ethereum needs to have a central set of computing rules, or else the machine wouldn't know what to do.
+
+Let's look at a couple of examples.
+
+### Example 1: United States Dollar coins {#usd-on-ethereum}
+
+For example, USDC is a United-States-Dollar-value cryptocurrency. It is "backed by fully reserved assets" (a phrase assumedly meant to communicate that it is fully backed by reserve assets). It is controlled in a relatively centralized way: via a permissioned consortium founded by two big corporations.
+
+DAI is another dollar-value coin. DAI gets created when people deposit crypto assets in a program (a smart contract). That smart contract was developed by a group called [MakerDAO](https://makerdao.com/en/). People who hold the MKR asset get to vote on how the program behaves. It is far more decentralized than USDC.
```

Comment:
May be interesting to make a note about recent changes allowing USDC as collateral to take out DAI loans, which arguably shifts the needle of DAI towards being more centralized. Could help to demonstrate that decentralization is not binary, but a spectrum, and different specific areas of the network can operate on different areas of that spectrum, with trade offs along the way.